pkg/assets/assets.go

```diff
@@ -0,0 +1,37 @@
+package assets
+
+import (
+	"bytes"
+	"text/template"
+)
+
+// Important: Run "make generate" to regenerate code after modifying/adding/removing any asset
+//go:generate go-bindata --prefix assets -pkg $GOPACKAGE -o bindata.go assets/...
+
+// SafeStringAsset Returns asset data as string
+// panic if not found or any err is detected
+func SafeStringAsset(name string) string {
+	data, err := Asset(name)
+	if err != nil {
+		panic(err)
+	}
+
+	return string(data)
+}
+
+// SafeStringAsset Returns asset data as string
```

Comment:
It seems it's an incorrect description (copy-paste from method above)?

Reply:
:+1: